pyproject.toml

```diff
@@ -57,8 +57,8 @@ dicompyler-core = { version = "*", optional = true } # groups = ["user", "all"]
 scikit-learn = { version = "*", optional = true }    # groups = ["user", "all", "mosaiq"]
 imageio = { version = "*", optional = true }         # groups = ["user", "all"]
 
-streamlit = { version = "~1.29.0", optional = true } # groups = ["user", "all"]
-streamlit-ace = { version = "*", optional = true }   # groups = ["user", "all"]
+streamlit = { version = ">=1.34.0", optional = true } # groups = ["user", "all"]
```

Comment:
I'd be more comfortable with ~1.34.0
streamlit does not seem quite that stable.

Reply:
Yup, that makes sense to me.

Reply:
```suggestion
streamlit = { version = "~1.34.0", optional = true } # groups = ["user", "all"]
```